routes.txt

```diff
@@ -0,0 +1,26 @@
+GET     /                                                       PageController :index
+GET     /sites                                                  SiteController :index
+GET     /sites/:id/edit                                         SiteController :edit
+GET     /sites/new                                              SiteController :new
+GET     /sites/:id                                              SiteController :show
+POST    /sites                                                  SiteController :create
+PATCH   /sites/:id                                              SiteController :update
+PUT     /sites/:id                                              SiteController :update
+DELETE  /sites/:id                                              SiteController :delete
+GET     /sites/:site_id/categories                              CategoryController :index
+GET     /sites/:site_id/categories/:id/edit                     CategoryController :edit
+GET     /sites/:site_id/categories/new                          CategoryController :new
+GET     /sites/:site_id/categories/:id                          CategoryController :show
+POST    /sites/:site_id/categories                              CategoryController :create
+PATCH   /sites/:site_id/categories/:id                          CategoryController :update
+PUT     /sites/:site_id/categories/:id                          CategoryController :update
+DELETE  /sites/:site_id/categories/:id                          CategoryController :delete
+GET     /sites/:site_id/categories/:category_id/posts           PostController :index
```

Comment:
posts should not be part via a cateogry. a post can have one or multiple categories. let's have them directly via /site/:site_id/posts